src/Sulu/Bundle/CategoryBundle/Resources/public/scss/modules/move-overlay.scss

```diff
@@ -0,0 +1,6 @@
+.move-categories {
+    .move-categories-container {
+        height: calc(100vh - 250px);
```

Comment:
TBH i have no idea why i have added this. the overlay handles the bigger container out of the box. i have removed it